src/app/shared/components/button/button.component.html

```diff
@@ -1,5 +1,7 @@
-<button class="button p-2 pe-3 ps-3"
-        [ngClass]="type.toLowerCase()"
-        [disabled]="isDisabled">
+<button class="button p-2 pe-3 ps-3" sofieTooltip [sofieTooltipTemplate]="tooltipRef" [ngClass]="type.toLowerCase()" [disabled]="isDisabled">
   <ng-content></ng-content>
 </button>
+
+<ng-template #tooltipRef>
+  <sofie-text-tooltip *ngIf="tooltip" [text]="tooltip"></sofie-text-tooltip>
```

Comment:
I think it would be nice to have the `*ngIf` inside of `sofie-text-tooltip` instead of here.
Then `sofie-text-tooltip` could take an optional string and only show it if it receives a non-empty string.
This way we don't need to write this if-statement everytime we need a text-tooltip some of the time.

Thoughts on the above is welcome.

Reply:
Fixed. I need to add one span to move css styles to this span, because `:host` elements is always visible and we apply black background and small padding (looks like a big black dot)